test/test_parser.rb

```diff
@@ -10054,6 +10054,15 @@ def test_private_endless_method_command_syntax
       SINCE_3_1)
   end
 
+  def test_value_omission
+    assert_parses(
+      s(:hash,
+        s(:pair, s(:sym, :a), s(:send, nil, :a)),
+        s(:pair, s(:sym, :b), s(:send, nil, :b))),
+      %q{{a:, b:}},
```

Comment:
There's a `source_maps` argument that has to be passed explicitly (`%q{}`).
Also, could you please add source maps checks for both key and value?